lib/rules/require-presentational-children.js

```diff
@@ -92,8 +96,11 @@ export default class RequirePresentationalChildren extends Rule {
         const children = getAllElementNodeDescendants(node);
 
         for (const child of children) {
+          // additional non semantic tags
+          const { additionalNonSemanticTags } = this.config;
+          const nonSemantic = new Set([...NON_SEMANTIC_TAGS, ...additionalNonSemanticTags]);
```

Comment:
Will this crash if a future option is passed and `additionalNonSemanticTags` is left as undefined?

Reply:
shouldn't crash, parseConfig handler above should cater for that